.storybook/main.js

```diff
@@ -28,6 +31,11 @@ module.exports = {
       },
       resolve: {
         ...resolve,
+        alias: {
```

Comment:
Look like there is a feature flag that allows fix that problem better way
https://github.com/iron-fish/ui-kit/pull/3/files#diff-cafe2123a72c4ce3a9f7e9ee4b0e188256eb02d7ec1e54fa2cadfac99b22f92bR34